astronomy/gravity_model.proto.txt

```diff
@@ -121,117 +146,213 @@ gravity_model {
+    angular_velocity        : "21.5710715 deg / d"
   }
   body {
     name                    : "Io"
     gravitational_parameter : "5.959916033410404e+03 km^3/s^2"
+    reference_instant       : 2451545.000000000
     mean_radius             : "1821.49 km"
+    axis_right_ascension    : "268.05 deg"
+    axis_declination        : "64.50 deg"
+    reference_angle         : "200.39 deg"
+    angular_velocity        : "203.4889538 deg / d"
   }
   body {
     # J2 coefficient and reference radius from DE431,
     # http://ilrs.gsfc.nasa.gov/docs/2014/196C.pdf.
     name                    : "Moon"
     gravitational_parameter : "4.9028000661637961e+03 km^3/s^2"
+    reference_instant       : 2451545.000000000
     mean_radius             : "1737.4 km"
     axis_right_ascension    : "269.9949 deg"
     axis_declination        : "66.5392 deg"
+    reference_angle         : "38.3213 deg"
+    angular_velocity        : "13.17635815 deg / d"
     j2                      : "2.0321568464952570e-04"
     reference_radius        : "1738.0 km"
   }
   body {
     name                    : "Europa"
     gravitational_parameter : "3.202738774922892e+03 km^3/s^2"
+    reference_instant       : 2451545.000000000
     mean_radius             : "1560.8 km"
+    axis_right_ascension    : "268.08 deg"
+    axis_declination        : "64.51 deg"
+    reference_angle         : "38.3213 deg"
+    angular_velocity        : "101.3747235 deg / d"
   }
   body {
     name                    : "Triton"
     gravitational_parameter : "1.427598140725034e+03 km^3/s^2"
+    reference_instant       : 2451545.000000000
     mean_radius             : "1352.6 km"
+    axis_right_ascension    : "299.36 deg"
+    axis_declination        : "41.17 deg"
+    reference_angle         : "296.53 deg"
+    angular_velocity        : "-61.2572637 deg / d"
   }
   body {
     # Gravitational characteristics from Brown and Schaller (2007),
     # "The Mass of Dwarf Planet Eris".  Radius from Sicardy et al. (2011),
     # "Size, density, albedo and atmosphere limit of dwarf planet Eris from a
-    # stellar occultation".
+    # stellar occultation".  Angular velocity from Roe et al. (2008), "Tentative
+    # detection of the rotation of Eris".
     name                    : "Eris"
     gravitational_parameter : "1108 km^3/s^2"
+    reference_instant       : 2451545.000000000
     mean_radius             : "1163 km"
+    axis_right_ascension    : "0 deg"  # Unknown.
+    axis_declination        : "0 deg"  # Unknown.
+    reference_angle         : "0 deg"  # Unknown.
+    angular_velocity        : "333 deg / d"
   }
   body {
     name                    : "Pluto"
     gravitational_parameter : "8.696138177608748e+02 km^3/s^2"
+    reference_instant       : 2451545.000000000
     mean_radius             : "1195 km"
+    axis_right_ascension    : "132.993 deg"
+    axis_declination        : "-6.163 deg"
+    reference_angle         : "237.305 deg"
+    angular_velocity        : "56.3625225 deg / d"
```

Comment:
NAIF says that they don't have data, JHU apparently didn't produce any Spice data.  Updated to 2013 or thereabout.